api/envoy/config/filter/network/direct_response/v2/config.proto

```diff
@@ -15,6 +17,6 @@ option (udpa.annotations.file_migrate).move_to_package =
 // [#extension: envoy.filters.network.direct_response]
 
 message Config {
-  // Response data as bytes.
-  bytes response = 1;
+  // Response data as a data source.
+  api.v2.core.DataSource response = 1;
```

Comment:
This is a breaking API change. You will need to add a new field for data source.

Reply:
This was added in the last few days. No one is using it yet so IMO it's fine to change.